lib/oli_web/live/community_live/show_view.ex

```diff
@@ -338,31 +343,50 @@ defmodule OliWeb.CommunityLive.ShowView do
     {:noreply, socket}
   end
 
-  def handle_event("add_institution", %{"name" => name}, socket) do
+  def handle_event(
+        "add_institution",
+        %{"institution_id" => ""},
```

Comment:
I'm wondering if we could delegate the error handling to the changeset? This could make the form more dynamic and give the responsibility to the changeset to control any errors send by the user.